src/validator/CreditCardChecking.java

```diff
@@ -0,0 +1,5 @@
+package validator;
+
+public interface CreditCardChecking {
+	public boolean checkValidity(CreditCardVendor ccv, String number);
```

Comment:
In this case methods are implicitly public.
No need to define it public.

I would suggest to remove the interface for now.
In final application you will expose REST API - and this will be the main API.
Maybe interface in this case is not needed since the Validator will be used only within service.